blog/## Features.md

```diff
@@ -0,0 +1,73 @@
+## Features
```

Comment:
Not sure if I'm missing anything here, but I think we probably don't want this file in the directory with all the blog posts.

Reply:
This was introduced in the `"lint"` commit acdc5c78cd6fb0a590c49b90fe5d46dc88e5f14f.